app/src/main/kotlin/com/wire/android/ui/home/conversations/search/HighLightName.kt

```diff
@@ -40,14 +40,12 @@ fun HighlightName(
     modifier: Modifier = Modifier
 ) {
 
-    val queryWithoutSuffix = searchQuery.removeQueryPrefix()
-
     val highlightIndexes = QueryMatchExtractor.extractQueryMatchIndexes(
-        matchText = queryWithoutSuffix,
+        matchText = searchQuery,
```

Comment:
when the user add @ it means to do search based on user handle, adding it to the search query will result to handle search being done with @ in the beginning 
for example my is a handle is "mo_wirer" then this is what will be stored in the database and I need to do handle search then the user enter "@mo_wire" and the actual search in the database will be against "@mo_wire" which will result to no valid results

Reply:
Totally agree, but the logic for search itself didnt change, we're still doing it as you explained, so if `@` is present it will be done like that, as it was before, this changes only the way its highlighted, so if we're looking for `@mat` and got results like:
- mat mat - wont highlight anything
- @mateusz.123 - will highlight @mat

The same goes if the query was done without `@` so if we search for `mat` it would be:
- mat mat - both maths are going to be highlighted
-@mateusz.123 - `mat` will get highlighted

Those changes will not affect any search results in any case

Reply:
ok got it